src/api/eventsAPI.js

```diff
@@ -697,7 +694,7 @@ function getEvents() {
 }
 
 function getEventById(id) {
-  // return axios.get('https://resistance-calendar.herokuapp.com/facebook/events')
+  // return axios.get('https://resistance-calendar.herokuapp.com/v1/events?$filter=_id eq \'' + id + '\'')
```

Comment:
Just as an FYI, one helpful feature in ES2015 is the addition of [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) which let us avoid string concatenation.